deploy/common/k8sbased.sh

```diff
@@ -83,10 +83,14 @@ EOF
 
 function verify_orch {
     if [ "$ORCH" == "openshift" ]; then
-        if kubectl api-versions | grep -q openshift.io; then
+        local out
+        out=$(kubectl api-versions 2>&1)
+        if echo "$out" | grep -q "openshift\.io"; then
```

Comment:
Is it debugging and will be removed?

Reply:
I'd suggest to leave it in. I have seen tests failing right here a few times and I would like to understand what is going on in case that happens again.

Reply:
could you please rename `out` to `k8s_api_version` or something like this

Reply:
done